File: Código/Loja/src/dados/Produto.java
package dados;

public class Produto {
    private String nome;
    private int codigo;
    private String cor;
    private double preco;
    private String marca;

    public Produto(String nome, int codigo, String cor, double preco, String marca) {
        this.nome = nome;
        this.codigo = codigo;
        this.cor = cor;
        this.preco = preco;
        this.marca = marca;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public int getCodigo() {
        return codigo;
    }

    public void setCodigo(int codigo) {
        this.codigo = codigo;
    }

    public String getCor() {
        return cor;
    }

    public void setCor(String cor) {
        this.cor = cor;
    }

    public double getPreco() {
        return preco;
    }

    public void setPreco(double preco) {
        this.preco = preco;
    }

    public String getMarca() {
        return marca;
    }

    public void setMarca(String marca) {
        this.marca = marca;
    }

    @Override
    public String toString() {
        return "Produto [nome=" + nome + ", codigo=" + codigo + ", cor=" + cor + ", preco=" + preco + ", marca=" + marca
                + "]";
    }

}

File: Código/Python/Loja/src/dados/Produto.py
class Produto:
    def __init__(self, nome, marca, modelo, cor, preco, quantidade):
        self.id = id =+ 1
        self.nome = nome
        self.codigo = None
        self.cor = cor
        self.preco = preco
        self.marca = marca
        self.modelo = modelo
        self.quantidade = quantidade

    def get_quantidade(self):
        return self.quantidade
    
    def get_nome(self):
        return self.nome
    
    def get_codigo(self):
        return self.codigo
    
    def get_marca(self):
        return self.marca

    def get_modelo(self):
        return
    def get_cor(self):
        return self.cor

    def get_preco(self):
        return self.preco
    
    def get_id(self):
        return self.id
    

    def __str__(self):
        return (f"Produto [id = {self.get_id}, nome={self.get_nome}, codigo={self.get_codigo}, cor={self.get_cor}, "
                f"preco={self.get_preco}, marca={self.get_marca}, modelo={self.get_modelo}]")